### ISSUE TITLE
Actions menu - Tabulation order in Firefox

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#1168]
We cannot access all options by tabulating in firefox:

![firefox_tabs](https://user-images.githubusercontent.com/7963804/45023112-1ad9bd00-b036-11e8-9a57-e3538dace33a.gif)

**Comment by matuzalemsteles** [comment]
hey @marcoscv-work I've been looking at this is seems to be a problem with MacOS and Firefox https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox (I've set up what people said about the problem and this works, just not a great approach...). 

We have to investigate whether even forcing the focus will work for Firefox.

**Comment by kresimir-coko** [comment]
Hey @marcoscv is this bug still reproducible?

**Comment by marcoscv-work** [comment]
We added the super-blue/white-double-focus, so it is no longer reproducible.

If sometime we found a new case where our focus is not working we can create a specific new task for adding it.

Thanks @kresimir-coko !
